include/reactor-cpp/scheduler.hh

```diff
@@ -27,123 +28,117 @@ namespace reactor {
 class Scheduler;
 class Worker;
 
-class Worker {
- public:
-  Scheduler& scheduler;
-  const unsigned id{0};
-  std::thread thread;
+class Worker { // NOLINT
+public:
+  Scheduler& scheduler_;
+  const unsigned int identity_{0};
+  std::thread thread_{};
 
   static thread_local const Worker* current_worker;
 
   void work() const;
   void execute_reaction(Reaction* reaction) const;
 
- public:
-  Worker(Scheduler& scheduler, unsigned id)
-      : scheduler{scheduler}, id{id}, thread{} {}
-  Worker(Worker&& w);
-  Worker(const Worker&) = delete;
+  Worker(Scheduler& scheduler, unsigned int identity)
+      : scheduler_{scheduler}
+      , identity_{identity} {}
```

Comment:
Also this syntax is different in the *.cc files. While here it lists `,` under `:` (which personally I prefer), but it in the *.cc files it was formatted like this: `: scheduler_{scheduler}, identity_{identity} {}`

Reply:
ok so u want it completely identically formatted okay. Understandable.